spec/models/user_spec.rb

```diff
@@ -6,6 +6,14 @@
   let(:successful_project){ create(:project, state: 'online') }
   let(:failed_project){ create(:project, state: 'online') }
   let(:facebook_provider){ create :oauth_provider, name: 'facebook' }
+  let(:staff_attributes) do
+    [
+      User.human_attribute_name('staff/team'),
```

Comment:
to ligado como o metodo funciona, qro saber como ta o arquivo de tradução pra precisar fazer esse negocio de `'staff/team'`